### ISSUE TITLE
Loadbalancer IP (MetalLB) not changeable anymore in Rancher 2.6 (GUI)

### ISSUE THREAD

**Issue by maveice** [rancher/dashboard#4110]
Last week I have started studying Rancher 2.6 and I really like it. 
Things are running great except one: 

Situation:
1. Rancher 2.6 ist installed and a new custom RKE-clsuter has been set up successfully
2. MetalLB with configmap is installed (with IP range) - automatic assignment of IPs is working well and as expected.

Problem:
In Rancher 2.5.9 (in the old GUI) while editing the deployment related loadbalncer service there was still an option "loadbalancer IP" available which I could easily fill in and change the loadbalancer IP (if needed) e.g. from 192.168.1.150 to 192.168.1.155 as long as it is part of the IP-range which I configured in MetalLB.

If you have an idea or approach how to handle this in Rancher v2.6 your help is very much appreciated! 😃
Many thanks in advance.

**Comment by nwmac** [comment]
@maveice Could you please provide a set of steps that you use in the 2.5.9 to set the load balancer IP. When you say "Old GUI", I assume you mean the Cluster Manager UI?

**Comment by maveice** [comment]
@nwmac - yes, "Old GUI" means "Cluster Manager UI" (in 2.5.9).

Steps: 
1. Pls. go to project "Default", then to the tab "Load Balancing"
2. Edit one of the loadbalancing services
3. At the bottom of the right hand side there is an option "Show advanced options" - click it
4. Please look for the field/box which is headlined "Load Balancer IP": You can insert your desired MetalLB IP there (as long as it is in the defined range)

Please let me know, if there are any further questions.

**Comment by davidnuzik** [comment]
**My checks PASSED**
 
**Reproduction Environment:**

Rancher version: v2.6.0
Rancher cluster type: single-node docker install
Docker version: 20.10
 
Downstream cluster type: n/a
Downstream K8s version: n/a
 
**Reproduction steps:**

1. Explore a cluster. Go to Service Discovery -> Services -> Create.
2. Create a Load Balancer
3. On the Create LoadBalancer form, click on "IP Addresses" and notice there is no field for Load Balancer IP any longer:
![image](https://user-images.githubusercontent.com/45179589/139955393-2e65ba51-6fc6-424f-94a0-a5fd8363928b.png)
 
-----
 
**Validation Environment:**

Rancher version: v2.6-head `931a0f09d` pulled at 11/1/21 ~1:15pm Arizona time
Rancher cluster type: single-node docker install
Docker version: 20.10
 
Downstream cluster type: n/a (does not matter)
Downstream K8s version: n/a (does not matter)
 
**Validation steps:**

1. Explore a cluster. Go to Service Discovery -> Services -> Create.
2. Create a Load Balancer
3. On the Create LoadBalancer form, click on "IP Addresses". Notice the Load Balancer IP field now exists :)
4. Successfully created a Load Balancer. Ensured the request/response to/from the server appeared correct.
![image](https://user-images.githubusercontent.com/45179589/139955980-e23d1bd3-aae0-44b1-a415-1d83c05c2e32.png)

Confirmed that the `services` POST request had content `loadBalancerIP` and was set to "192.168.0.2" in the spec. The response looks normal.
Edited this service (the lb entry). Confirmed in the UI I still see my Load Balancer IP address. I modified this to "192.168.0.3" and saved. A PUT request for `test` (the name of my service) was sent successfully with the modified `loadBalancerIP` in the spec. This response also looks normal.

**Additional Info:**
I do not have a suitable means to set up a custom cluster with MetalLB in a reasonable amount of time to test this with such a setup/scenario; however, considering this was just a frontend change my validation of the field and confirmation that edits (PUT requests) works as expected should be sufficient.
